### ISSUE TITLE
Dashboard: Revisions tab - French language link sends users to a Gateway Timeout page

### ISSUE THREAD

**Issue by XiuJuanXia** [mmiron-HC/WST-RSAMS#116]
URL: https://hcdev.openplus.ca/en/node/60168/revisions/view/591938/597590/visual_inline
User Agent: Chrome
Language: English

Issue Description: French language link sends users to a Gateway Timeout page.
![image](https://user-images.githubusercontent.com/88776404/137137523-ba45f6ab-fc8f-45f7-a374-a39a4519f397.png)

Steps to reproduce:

1. Log into the Dashboard
2. Action on the "Build"
3. Select "Revisions" tab
4. To compare two revisions, use the radio buttons from column A and B and press the compare button
5. Selecting French language link
6. Users see a Gateway Timeout page

Expected results: Appropriate French page should be directed when users select the French language link.

**Comment by davidgotlieb** [comment]
Working correctly on the live site now

**Comment by davidgotlieb** [comment]
Fixed on live site

**Comment by XiuJuanXia** [comment]
Issue no longer exists as the Francais link is removed from the revision tab. Closing this issue.
